src/GeekLearning.DotNet.Swashbuckle/Program.cs

```diff
@@ -62,13 +41,12 @@ public static void Main(string[] args)
             var generationContext = new
             {
                 ProjectPath = projectFile,
-                //IsNetCoreApp = hasNetCoreAppFramework && (options.Framework == null || options.Framework.Framework.StartsWith(netCoreAppFramework)),
                 TargetFramework = properties[targetFrameworkProperty],
                 AssemblyName = properties[assemblyNameProperty],
                 ContentRoot = projectDirectory,
-                ApiVersion = options.ApiVersion,
+                options.ApiVersion,
```

Comment:
this is disgusting @asiffermann

Reply:
Hmmm, not sure you want to open a discussion on styling code... 😛
On this particular point, I just followed the VS recommandation. If we don't want it, we should provide an .editorconfig file in our repos.